easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '7.11.6'
```

Comment:
current last public release is 7.10.4, so we should add another easyconfig for that?

http://www.abinit.org/downloads/source-packages/abinit-1